### PR TITLE
LinkedTrust team edits of the LinkedClaims proposal

### PR DESCRIPTION
**Pull request by gvelez17**
Editing the summary, motivation and deliverables for 2 purposes:

1) to clarify the work to be done during the lab

2) to invite stakeholders by pointing to a document in this folder

### CODE REVIEW

**Comment by gvelez17** [comment]
This is still a draft we are going to edit it a bit more today, i don't see a way to mark as draft in this fork

**Comment by andorsk** [comment]
looks like you got this as a draft. looking forward to seeing the proposal. you'll be the first :)

**Comment by gvelez17** [comment]
hey @andorsk ready to merge!  we wound up using all the time for internal review since we have a small group contributing to this.  Looking forward to working with you all on this now!

**Comment by andorsk** [comment]
awesome edits. Merging in!

File: proposals/linked_claims/001_proposal.md [hunk 1]
```diff
@@ -17,50 +17,77 @@
...
 The intention is to have this be a lightweight open standard.
 
 ## Unresolved Questions
 
+- Is there a need for a specific "glue" vocabulary such as http://cooperation.org/credentials/v1/
+- How to resolve the tension between using a human-viewable subject URI versus a hashable signed credential as the subject of a claim?  Could there be a content type or query parameter that switches, but in this case how to ensure the two are aligned?
+- Should the canonical identifier of a claim be included in the claim itself?
+- Review each of the MUST, SHOULD and MAY in the RFC for usefulness and testability
+- Can we write tests for being a LinkedClaim without a single canonical vocabulary?  
+
```
Comment (andorsk) [inline proposals/linked_claims/001_proposal.md:84]: nit: extra line.

File: proposals/linked_claims/001_proposal.md [hunk 2]
```diff
@@ -17,50 +17,77 @@
...
+and the largest question
+
```
Comment (andorsk) [inline proposals/linked_claims/001_proposal.md:86]: nit: extra line

**Review by andorsk — COMMENTED** [review]
great work. All the edits are reasonable.

**Pull request merged**